gallery/src/styles/DarkMode/DarkModeDarkOverrideExample.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML>
+<html class="nx-html nx-html--page-scrolling nx-html--enable-color-schemes nx-html--dark-mode">
+  <head>
+    <title>Dark Mode Example: No Opt-In</title>
```

Comment:
This example does opt-in to dark mode

Reply:
> This example does opt-in to dark mode (same for DarkModeLightOverrideExample and DarkModeOptInExample)

Fixed all of the copy/pasted example titles in 843efbafa